frontend/pages/index.vue

```diff
@@ -17,6 +17,7 @@
         </div>
         <div class="site__main-company py-2 md:py-8">
           <Company />
+          <Average title="Average" salary="150 000 FCFA" />
```

Comment:
hi Assam Thank one more time for the Amazing work.

why is the average salary fixed to 150 000?? should we not compute that from the list of salaries??

i will suggest you to make a computed property for it and the pass the computed value to the composant so that we will have a real average of the salaries.

Reply:
The value of the average salary will be calculated on the side of the back end. So we will have a specific endpoint for this

Reply:
ok as i get the explanation it is also okay for me.